File: app/src/main/java/net/tlalka/android/fiszki/activities/HelpActivity.java
package net.tlalka.android.fiszki.activities;

import android.os.Bundle;
import android.view.View;
import net.tlalka.android.fiszki.R;

public class HelpActivity extends AbstractActivity {

    @Override
    public void onCreate(Bundle bundle) {
        super.onCreate(bundle);
        super.setContentView(R.layout.help_view);
    }

    public void onViewClick(View view) {
        super.finish();
    }
}

File: app/src/main/kotlin/net/tlalka/fiszki/view/activities/HelpActivity.kt
package net.tlalka.fiszki.view.activities

import android.os.Bundle
import android.view.View
import butterknife.OnClick
import net.tlalka.fiszki.R

class HelpActivity : BasePageActivity() {

    public override fun onCreate(bundle: Bundle?) {
        super.onCreate(bundle)
        super.setContentView(R.layout.help_activity)
        super.activityComponent.inject(this)
    }

    @OnClick(R.id.help_layout)
    fun onViewClick(@Suppress("UNUSED_PARAMETER") view: View) {
        super.finish()
    }
}
